IdentityCore/src/requests/broker/MSIDBrokerTokenRequest.m

```diff
@@ -178,17 +178,16 @@ - (NSDictionary *)defaultPayloadContents:(NSError **)error
 
 - (NSDictionary *)defaultResumeDictionaryContents
 {
-    NSDictionary *resumeDictionary =
-    @{
-      @"authority"        : self.requestParameters.authority.url.absoluteString,
-      @"client_id"        : self.requestParameters.clientId,
-      @"redirect_uri"     : self.requestParameters.redirectUri,
-      @"correlation_id"   : self.requestParameters.correlationId.UUIDString,
+    NSMutableDictionary *resumeDictionary = [NSMutableDictionary new];
```

Comment:
is the purpose of this change to ensure we don't crash if one of the parameters is nil?